pkg/www/retext/src/components/nav/ToDocument.vue

```diff
@@ -22,10 +22,10 @@
 
     export default {
         name: "ToDocument",
-        props: ["documentName", "documentId", "path", "buttonText"],
+        props: ["documentName", "documentId", "path", "buttonText", "projectId"],
         methods: {
             goto() {
-                this.$router.push(`${this.path}/${this.documentId}`)
+                this.$router.push(`/project/${this.projectId}${this.path}/${this.documentId}`)
```

Comment:
Opened a new issue for this, since I don't love the way I did it, but its also a large change, while the current stuff works (for now)

#49 Improve bread/crumb routing logic #49